src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -232,8 +236,7 @@ public void setUp() throws IOException, COSVisitorException {
     @After
     public void tearDown() {
         if (!pdfFile.delete()) {
-            System.err
-                    .println("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
+            LOGGER.info("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
```

Comment:
Better use TemporyFolder as described in the wiki https://github.com/JabRef/jabref/wiki/Code-Howtos#files-and-folders-in-tests